deeplabcut/utils/auxfun_videos.py

```diff
@@ -42,6 +42,15 @@ def check_integrity(self):
         if os.path.getsize(dest) != 0:
             warnings.warn(f'Video contains errors. See "{dest}" for a detailed report.')
 
+    def check_integrity_robust(self):
+        numframes = self.video.get(cv2.CAP_PROP_FRAME_COUNT)
```

Comment:
Since opencv can also return the wrong number of frames with corrupted metadata, do you feel strongly about the following?
```python
while self.video.isOpened():
   ...
```